ddtrace/settings/config.py

```diff
@@ -552,6 +552,7 @@ def __init__(self):
 
         self._llmobs_enabled = asbool(os.getenv("DD_LLMOBS_ENABLED", False))
         self._llmobs_sample_rate = float(os.getenv("DD_LLMOBS_SAMPLE_RATE", 1.0))
+        self._llmobs_ml_app = os.getenv("DD_LLMOBS_ML_APP", "")
```

Comment:
If we don't expect customers to set this envar we should prefix it with an underscore
```suggestion
        self._llmobs_ml_app = os.getenv("_DD_LLMOBS_ML_APP", "")
```

Reply:
Good point - I'll just remove this from the config for now, as users might use it but I don't want to visibly add this to our public API.